tools/debugging/mint.sh

```diff
@@ -23,7 +23,9 @@ require_bin parallel
 
 [ $# -lt 2 ] && die "${0} <token_address> <raiden_server>+"
 
+# shellcheck disable=SC2034
 UINT256_MAX=115792089237316195423570985008687907853269984665640564039457584007913129639935
+# shellcheck disable=SC2034
 UINT128_MAX=340282366920938463463374607431768211455
```

Comment:
Do we keep these in for future use, or are they actually used anywhere?

Reply:
Yeah I wasn't sure either. I assumed they are defined in case a user wants to try different values...